README.md

```diff
@@ -4,6 +4,8 @@
 
 Myproject is a demo of webpage catalog using html and css. Is built with mobile first best practices.
 
+To check online visit: [My Project](https://gjuliao.github.io/myproject/)
+
 **Built with**
 
 ---
```

Comment:
:red_circle:  The online version of your portfolio looks great :+1:  But one of your project images (on the third project card) is not showing. Kindly make sure you type the correct image path. I think you have forgotten to add a dot `.` before `/`
Kindly check this image on the third project card :arrow_down: 

![screenshot(13)](https://user-images.githubusercontent.com/26736582/200364231-784d9e2c-69b8-4cd7-9170-d58894b0a0c6.png)